src/OrchardCore.Modules/OrchardCore.Tenants/Views/Shared/_TenantDatabaseInfo.cshtml

```diff
@@ -0,0 +1,49 @@
+@model EditTenantViewModel
```

Comment:
I'm not sure why the partials relying on `EditTenantViewModel`, it would be nice to use the UI components in both Create & Edit views